backend/backend/services/business_logic.py

```diff
@@ -13,7 +13,7 @@ async def get_test_result(context: Context, subject: str):
     summary_chain = context.chains.get_chains()['summary_chain']
```

Comment:
The loop condition should be changed from range(2) to range(1) to ensure the function iterates over all possible values.